05_stringer/kasunfdo/main_test.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote("127.0.0.1\n")
+	actual := strconv.Quote(buf.String())
```

Comment:
Don't understand why use `strconv.Quote` here

Reply:
Although this still works but yeah you could have less code. Yeah other than this it looks good.

Reply:
Changed. Thank you!